performance_test/src/idlgen/opendds/CMakeLists.txt

```diff
@@ -0,0 +1,115 @@
+# Copyright 2017 Apex.AI, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set(SOURCE_EXTENSION cpp)
+
+set(DDS_ROOT /opt/opendds/share/dds)
```

Comment:
It would be good to be able to set this with an environment variable or something